### ISSUE TITLE
What is Refinery?

### ISSUE THREAD

**Issue by Deozaan** [itchio/itch.io#450]
Am I the only one confused by Refinery? The website and announcements and documentation make it sound like Refinery is "a thing" or a tool you can use, but it's really just a collection of disparate features on itch.io that aren't even really organized together in a single section of the dashboard. I'm still unsure of how to "use" Refinery.

Not only that, but some of the features seem to be inexplicably linked to other unrelated features, such as limiting the number of keys being linked to exclusive content which seems to be only for people who pay money for the content (similar to different Kickstarter backer tiers).

But what if I wanted to use Refinery/Itch.io as a way to organize and get feedback on a limited, but free beta? As far as I can tell, the only way for me to do that would be to manually make a limited number of download keys and track and distribute them manually. That's quite a hassle. I'd much rather be able to link to the page on itch.io and let people sign up for the beta until the limit is reached. And maybe later increase the limit so that more people can sign up later.

Anyway, that's getting a bit off topic. Mostly I'm just bewildered by Refinery, what it is, and how to use it. Because like I said, it's talked about as if it's a single monolithic entity, but in all my research I can only come to the conclusion that it's really just to a disorganized idea (or workflow), using different features available on the website and through the command line tools.

**Comment by fasterthanlime** [comment]
> But what if I wanted to use Refinery/Itch.io as a way to organize and get feedback on a limited, but free beta?

That sounds like the actionable part of the issue 😉 and it doesn't even sound that hard for us to add!

**Comment by Deozaan** [comment]
Yeah. The fact that I can't set up a free, limited beta as described earlier has added to my confusion on refinery. And getting such a feature added is definitely something I want, but I consider it a different issue.

This issue is meant more as feedback about the confusing nature of how refinery is "advertised" as a tool that you can use, when in practice it seems to be a workflow involving itch.io's various features, most of which have existed long before refinery was announced.

The announcement(s) and documentation give the impression that refinery is an API or program. A "thing" people can use for distributing and forming a community around pre-release games. But other than [the refinery page](http://itch.io/refinery), refinery isn't even mentioned anywhere in the developer/creator dashboards or "edit game" pages that I can find, and possibly not even anywhere else on the site.

What this means is that if someone hears about refinery (like I did), thinks it's a wonderful idea (like I do), and wants to get set up to "use refinery" on their project (like I do), they're not going to know where to go or how to do that (like I don't).

And it's difficult to explain to someone else how to use refinery because the features of refinery in the documentation aren't labelled as being part of refinery, and are spread out all over the place.

Here's what I think is the main actionable part to resolving the issue: I think this problem of not knowing what refinery is or how to use it would be solved if there was a "refinery" tab on each game's dashboard with all the refinery features (or links to them) in one place. A single place I could go to while editing my game(s) to enable and adjust refinery's features.

**Comment by fasterthanlime** [comment]
Did you perchance miss that link? https://itch.io/docs/creators/refinery
> Here's what I think is the main actionable part to resolving the issue: I think this problem of not knowing what refinery is or how to use it would be solved if there was a "refinery" tab on each game's dashboard with all the refinery features (or links to them) in one place. A single place I could go to while editing my game(s) to enable and adjust refinery's features.

Revamping the edit game page was in our refinery plans but we eventually ran out of time. @leafo is still polishing everything we announced during #itchioweek. I'm not sure if that is still in his plans but I agree it'd be nice! :+1:

**Comment by Guard13007** [comment]
Just poking around issues right now, I'm guessing this one can be safely closed. The only thing still open about it was made into its own issue.
